android/daemon/src/main/java/org/mozilla/firefox/vpn/daemon/VPNService.kt

```diff
@@ -254,6 +255,15 @@ class VPNService : android.net.VpnService() {
         }
 
         if (isSuperDooperMetricsActive) {
+            val installationIdString = json.getString("installationId")
+            installationIdString?.let{
+                try {
+                    val installationId = UUID.fromString(installationIdString)
+                    Session.installationId.set(installationId)
+                } catch (e: Exception) {
+                    Log.e(tag, "Daemon installation ID string was not UUID")
```

Comment:
nit: Let's also add the exception message to this log.

Reply:
Good call, done. Thanks.